### ISSUE TITLE
Point cloud library duplication

### ISSUE THREAD

**Issue by Russ76** [MobileRobotics-Ulaval/husky-trainer#7]
Thanks for the answer. Furthermore:
The ZED-ROS-wrapper requires the huge and costly PCL routine. I wonder if libpointmatcher isn't doing already some of these same things; so both wouldn't be needed. As you say, performance must always be considered.

Here is their helpful answer from answers.ros.org.
Airuno2L  answered: 
I've never used the Zed wrapper but looking at the github page and going to the v1.0.0 tag, it looks like version 1.0.0 works with Indigo (it's mentioned in the README.md). 
I know that the Husky doesn't have support yet for Kinetic from this question here. So I think your best option is to install Indigo and try that v1.0.0 of the Zed wrapper. Even if Clearpath did finish the Husky's support for Kinetic soon, who knows if MobileRobotics-Ulaval will update husky-trainer. 
https://github.com/stereolabs/zed-ros-wrapper/tree/v1.0.0

In reply to Rodolfo8/(russh@perfectsecurity.us) 's question: 
Which version of ROS is needed?
Tags: zed_camera, jetson_tx1, clearpath_husky. 
https://github.com/stereolabs/zed-ros...
https://github.com/MobileRobotics-Ula...
The first package specifies Ubuntu 16.04 and ROS Kinetic. The second specifies ROS Hydro or Indigo. (and Ubuntu 14.04)
Does this mean it's impossible to use them together?
In other words: I was hoping to use Husky-trainer (on a different machine, similar to the Husky) with the ZED stereo camera (and no laser) but I see ZED's latest ROS packages specify Ubuntu 16.04 and ROS Kinetic. It seems ROS Indigo wants Ubuntu 14.04. I am using the NVIDIA Jetson TX1.

**Comment by davidlandry93** [comment]
I can confirm there is no project to port husky-trainer to Kinetic over here. 

Note that even if this project is called husky-trainer there is on direct dependency on anything by Clearpath. Although this was never tested, the code should be directly applicable to any differential drive robot (with an appropriate launchfile). This means you don't have to wait for Clearpath to upgrade anything to Kinetic.

As for requiring only one of PCL and libpointmatcher, I don't think it's likely that you can easily pull PCL out of the zed wrapper. But who knows! However, please reserve this repository for issues directly concerning the husky-trainer code ;)

**Comment by Russ76** [comment]
Thank you, David!
